### PR TITLE
 Triangle().area for zero-area triangle results in AttributeError 

### LINKED ISSUE #20460: Triangle().area for zero-area triangle results in AttributeError
`Triangle(Point(0,0),Point(0,1),Point(0,0.5)).area`
results in 
`---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-11-4686e8494747> in <module>
----> 1 Triangle(Point(0,0),Point(0,1),Point(0,0.5)).area

AttributeError: 'Segment2D' object has no attribute 'area'`

I'm new to the project and understand that there may be some good reasons to cast a zero-area Triangle to a Segment2D; I would also strongly argue that Triangle() and its functions should work correctly for arbitrary user-inputted or randomly-generated points. Perhaps a simple fix is to implement Segment2D.area and any other functions for finite-area polygons?

### PR DESCRIPTION
**Pull request by ghost**
#### References to other Issues or PRs

Fixes #20460 

#### Brief description of what is fixed or changed
The Point and Linear Enitity didn't had  an area attribute which resulted in an Attribute Error whenever someone tried to access their area. 
Added tests for checking these changes.

#### Other comments
[this](https://github.com/sympy/sympy/pull/20469) didn't completely fix the problem since it only added  the area attribute to Segment2D and not all the lines in n dimensional space.

#### Release Notes

- geometry
  - Now Lines and points  have an area attribute .

### CODE REVIEW

**Comment by sidhu1012** [comment]
`tests` shouldn't go in release notes. Check  [guide](https://github.com/sympy/sympy/wiki/Writing-Release-Notes)

File: sympy/geometry/tests/test_line.py [hunk 1]
```diff
@@ -271,8 +272,10 @@ def test_basic_properties_3d():
     assert p1 not in l3
 
     assert l1.direction_ratio == [1, 1, 1]
+    assert l1.area==0
 
     assert s1.midpoint == Point3D(S.Half, S.Half, S.Half)
+
```
Comment (sidhu1012) [inline sympy/geometry/tests/test_line.py:278]: Remove blank line, it's causing test to fail.

**Comment by sidhu1012** [comment]
It would be better to add `area` attribute to `class LinearEntity` and all other linear entity classes . It would make changes symmetric and would give `area = 0` to all lines.

**Comment by almighty123789** [comment]
@sidhu1012  but doens't all other linear entities (line, ray, segment ) inherit from the Linear Entity class ?   I have added this `area` attribute to the LinearEntity and the Point Class.

**Comment by sidhu1012** [comment]
> @sidhu1012 but doens't all other linear entities (line, ray, segment ) inherit from the Linear Entity class ? 

That's the point. If we are adding `area` attribute to line and point , then it would be better that all `Linear Entity` has this area attribute which gives `0` and it would be wise to add that in abstract class.

File: sympy/geometry/line.py [hunk 2]
```diff
@@ -51,10 +51,12 @@ class LinearEntity(GeometrySet):
     ambient_dimension
     direction
     length
+    area
     p1
     p2
     points
 
+
```
Comment (sidhu1012) [inline sympy/geometry/line.py:59]: Why this blank line?

Comment (sidhu1012) [inline sympy/geometry/line.py:59]: Why this blank line?

**Comment by sidhu1012** [comment]
New commits add changes to irrelevant files.I think you pushed them by mistake. Please revert them as they add new files.

**Pull request closed without merging**